README.md

```diff
@@ -1,46 +1,59 @@
-# Getting Started with Create React App and Redux
+# Fruity App
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app), using the [Redux](https://redux.js.org/) and [Redux Toolkit](https://redux-toolkit.js.org/) template.
+A fruits information app where you can see the nutritional values of several fruits.
 
-## Available Scripts
+# Built With
 
-In the project directory, you can run:
+- Major Languages
 
-### `npm start`
+  - JavaScript
+  - React
+  - CSS
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+- Initializer
+  - create-react-app
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+## Getting Started
 
-### `npm test`
+- Clone this repo
+- Install dependencies
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+```
+npm install
+```
 
-### `npm run build`
+- Run build script
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+```
+npm run build
+```
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+- You can find the built web app on ./dist
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+- Alternatively you can start the development server
 
-### `npm run eject`
+```
+npm start
+```
 
-**Note: this is a one-way operation. Once you `eject`, you can't go back!**
+# Authors
 
-If you aren't satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+**Fábio Vinícius Dias Lima**
 
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you're on your own.
+- GitHub: (https://github.com/limavfabio)
 
-You don't have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn't feel obligated to use this feature. However we understand that this tool wouldn't be useful if you couldn't customize it when you are ready for it.
+# Contributing
 
-## Learn More
+- Please check the issues page
 
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+# Live Demo
```

Comment:
- [ ] As requested by the previous reviewer, please ensure to deploy your application using Heroku, Netlify, or GitHub Pages. kindly ensure that the project is deployed correctly, the home and detail page works as expected. Then attach a link to the deployed project to this part of the README